ui/app/manager/src/pages/page-map.ts

```diff
@@ -99,7 +101,40 @@ export function pageMapProvider(store: EnhancedStore<MapStateKeyed>, config?: Pa
         ],
         pageCreator: () => {
             const page = new PageMap(store);
-            if(config) page.config = config;
+
+            page.config = config || {};
+
+            // todo: remove stub
```

Comment:
Should get rid of this as it should come through the manager from a config file set by the user.